portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -41,50 +42,33 @@ public Login(String status, String url) {
             this.url = url;
         }
     }
+
+    private static final String urlToRedirectToAfterUserLogsIn = "/nickname.html";
```

Comment:
Best practice for constants is to use `SCREAMING_SNAKE_CASE`

Reply:
done